docs/index.md

```diff
@@ -1,215 +1,53 @@
 ---
 hide:
   - toc
+  - navigation
+  - footer
 ---
 
-# :ibis-logo: Ibis
+# <span style="font-size: 1.5em; margin: 0">:ibis-logo: The Ibis Project</span>
 
-## Expressive analytics in Python at any scale.
+## Data analytics at scale
 
-<script 
-    src="https://asciinema.org/a/yp5Ww4XKyjJsUCXkEz5or9rPq.js"
-    data-autoplay="true"
-    data-preload="true"
-    data-loop="true"
-    data-i="4"
-    data-rows="20"
-    id="asciicast-yp5Ww4XKyjJsUCXkEz5or9rPq"
-    async>
-</script>
+The flexibility of Python with the scale and performance of modern SQL.
 
-## Installation
+[Install](/install){ .md-button .md-button--primary }
```

Comment:
Does not exist; see https://github.com/ibis-project/ibis/pull/4768/files#r1016875012